scalableminds/webknossos
# Extracted strings in login/register for customization

Steps to test:
- Check that login and register messages still show the same by default
- Set config `customMessages.login` messages and see if it changed

Issues:
- contributes to #1608

------
